controller_definition.py

```diff
@@ -418,7 +418,7 @@ def OnDirtyMixerTrack(self, index):
     def OnUpdateMeters(self):           # Intended to be declared by child
         raise NotImplementedError()
 
-    def adjustMixer(self, knob: int, dataType: str, action: str, selectedTrack: int):
+    def adjustMixer(self, knob: int, dataType: str, action: str, selectedTrack: int, sensitivity: float = None):
```

Comment:
`sensitivity: float = None` should be `speed: int` instead. Also, please comment the parameter in the docstring of the function.

Reply:
I pushed https://github.com/hobyst/flmidi-kompletekontrol/pull/20/commits/9094dd12d836bd7f538ba522cae24c02afaf4f17, which should resolve this. Let me know if I can elaborate more on the description I put.